src/lib/utils/cuckoo_hashtable.hpp

```diff
@@ -57,9 +57,9 @@ class HashTable : private Noncopyable {
   std::shared_ptr<PosList> get(S value) {
     for (size_t i = 0; i < NUMBER_OF_HASH_FUNCTIONS; i++) {
       auto position = hash<S>(i, value);
-      auto elem = _hashtables[i][position];
-      if (elem != nullptr && value_equal(elem->value, value)) {
-        return elem->row_ids;
+      auto element = _hashtables[i][position];
+      if (element != nullptr && value_equal(element->value, value)) {
+        return element->row_ids;
```

Comment:
I think most of the changes make sense. IMHO, `elem` is clear enough, though. At another place, you changed `iter` to `it`, which is even shorter.

Reply:
`_it` is used in most places. I am fine with either, but this should also be consistent.

Reply:
Okay should this and `left` / `right` be added to the glossary?

Reply:
Added `_it`. I don't think that left/right fits.